deps/src/main/java/com/microsoft/azure/sdk/iot/deps/serializer/DeviceParser.java

```diff
@@ -196,193 +200,180 @@ public DeviceParser(String json)
     }
 
     /**
-     * Getter for moduleId
+     * Getter for ModuleId.
```

Comment:
I updated this file to have consistent doc comments, including consistent casing.